source/_integrations/shelly.markdown

```diff
@@ -208,13 +208,15 @@ Depending on how a device's button type is configured, the integration will crea
 
 ### Binary input sensors (generation 2+)
 
-For generation 2+ hardware, it's possible to select if a device's input is connected to a button or a switch. Binary sensors are created only if the input mode is set to `switch`. When the input is of type `button` you need to use events for your automations.
+For generation 2+ hardware, it's possible to select if a device's input is connected to a button or a switch. Binary sensors are created only if the **Input Mode** is set to `Switch`. When the **Input Mode** is set to `Button` you need to use events for your automations.
 
-## Event entities (generation 1)
+## Event entities
 
-If the **BUTTON TYPE** of the switch connected to the device is set to `momentary` or `detached switch`, the integration creates an event entity for this switch. You can use this entity in your automations.
+### Event entities (generation 1)
 
-## Event entities (generation 2+)
+If the **Button Type** of the switch connected to the device is set to `momentary` or `detached switch`, the integration creates an event entity for this switch. You can use this entity in your automations.
```

Comment:
```suggestion
If the **BUTTON TYPE*** of the switch connected to the device is set to `Momentary` or `Detached Switch`, the integration creates an event entity for this switch. You can use this entity in your automations.
```

For Gen1 this is how it looks:
<img width="817" height="349" alt="image" src="https://github.com/user-attachments/assets/4e9c78e4-e45d-4796-a251-cfd4176504d7" />